src/app/core/metadata/metadata.service.ts

```diff
@@ -199,8 +209,8 @@ export class MetadataService {
    * Add <meta name="citation_author" ... >  to the <head>
    */
   private setCitationAuthorTags(): void {
-    const values: string[] = this.getMetaTagValues(['dc.author', 'dc.contributor.author', 'dc.creator']);
-    this.addMetaTags('citation_author', values);
+    const values: string = this.getFirstMetaTagValue(['dc.author', 'dc.contributor.author', 'dc.creator']);
```

Comment:
why only the first one?

Reply:
Because here in clarin they have it as only first:
https://github.com/ufal/clarin-dspace/blob/8780782ce2977d304f2390b745a98eaea00b8255/dspace/config/crosswalks/google-metadata.properties#L41

([Pipe delimited lists explanation](https://github.com/ufal/clarin-dspace/blob/8780782ce2977d304f2390b745a98eaea00b8255/dspace/config/crosswalks/google-metadata.properties#L21)